### ISSUE TITLE
Add CXX and C flags options

### ISSUE THREAD

**Issue by johnwason** [MarkusJx/googletest-installer#1]
I need to pass -fPIC to the google test build. Can CXX and C flags options be added?

**Comment by MarkusJx** [comment]
You should now be able to pass C and C++ flags to the build process:
```yml
- name: Install gtest
  uses: MarkusJx/googletest-installer@93885edf173c4d59c40b3fa7d851b0be69967bd0
  with:
    C_FLAGS: YOUR_C_FLAGS
    CXX_FLAGS: YOUR_CPP_FLAGS
```

Just let me know if that worked for you and I'll draft a new Release or adapt the code if required :)

**Comment by MarkusJx** [comment]
Can you please provide feedback if this is working, so I can close this issue or adapt the action further

**Comment by johnwason** [comment]
Sorry, haven't had a change to look at this yet. I will try to get to it this week.

**Comment by johnwason** [comment]
I am seeing a failure in Ubuntu 16.04:

https://github.com/johnwason/robotraconteur/actions/runs/3932509904/jobs/6725168287

**Comment by MarkusJx** [comment]
May be fixed by the latest tag: `6dc68cc0073e5696daaf921dfc27a69f3e9659df `

```yml
- name: Install gtest
  uses: MarkusJx/googletest-installer@6dc68cc0073e5696daaf921dfc27a69f3e9659df
  with:
    C_FLAGS: YOUR_C_FLAGS
    CXX_FLAGS: YOUR_CPP_FLAGS
```

**Comment by johnwason** [comment]
Still lots of errors, this time the error happens with more versions.

https://github.com/johnwason/robotraconteur/actions/runs/4040694603/jobs/6946576094

I will use a manual build.
